components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -554,7 +554,7 @@ - (instancetype)init {
     _uppercaseButtonTitle = YES;
     _disabledButtonAlpha = (CGFloat)0.12;
     _messageElevation = MDCShadowElevationSnackbar;
-    _adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable = YES;
+    _adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable = NO;
```

Comment:
I think it's worth splitting this particular change (the default + deprecation) to a separate PR.